src/operators/ops_regular_cartesian_grid.jl

```diff
@@ -206,13 +206,8 @@ end
     (δx_e2f_ūᶻw̄ˣ(g, u, w, i, j, k) / g.Δx) + (δy_e2f_v̄ᶻw̄ʸ(g, v, w, i, j, k) / g.Δy) + (δz_c2f_w̄ᶻw̄ᶻ(g, w, i, j, k) / g.Δz)
 end
 
-@inline function Gu_cori(g::RegularCartesianGrid, v, f, i, j, k)
-    f*( avgy_f2c(g, v, decmod1(i, g.Nx), j, k) + avgy_f2c(g, v, i, j, k) )*0.5
-end
-
-@inline function Gv_cori(g::RegularCartesianGrid, u, f, i, j, k)
-   -f*( avgx_f2c(g, u, i, decmod1(j, g.Ny), k) + avgx_f2c(g, u, i, j, k) )*0.5
-end
+@inline function Gu_coriolis(g::RegularCartesianGrid, v, f, i, j, k) =  0.5 * f * (avgy_f2c(g, v, decmod1(i, g.Nx), j, k) + avgy_f2c(g, v, i, j, k))
```

Comment:
I'd actually prefer a more mathematical name for this function (similar to the other terms in the U-equation, something like `f_×_u`.

Note that the '0.5' are still occuring here. Also, I'm not sure this function is specific to a `RegularCartesianGrid` --- won't it be valid for all `Grid`s?

Reply:
+1. Unfortunately we can't use `f×u` as a function name :( I'll go with `fxu` and `fxv` unless there are any better ideas.

Yeah I'm still not converting the `0.5`, I figured we can do all of them in a future PR.

Haven't worked it out but this function might be specific to `RegularCartesianGrid`, or at least to `CartesianGrid`, as you'll get area and volume factors otherwise. But yeah it works for `RegularCartesianGrid` and `VerticallyStretchedCartesianGrid` or whatever we wanna call it.

Reply:
Ah actually `fxu` and `fxv` would be wrong as you don't cross product with the individual scalar velocity fields. I'll think of something else.

Reply:
What do you think of `fv(...)` for the _u_-momentum equation and `fu(...)` for the _v_-momentum equation?

Then we'd have to take the minus sign out of the `fu` function and put it in the source term calculation kernel so it shows up as `-fu(...)`. It'll look more like the blackboard math but might be confusing?

Reply:
I think that the more it looks like the blackboard math, the *less* confusing it is. I'd be curious to know if other's agree.

Note that the Coriolis terms that appears in each momentum equation are the x- and y-components of the *vector* Coriolis term `𝐟 ⨲ 𝐮`.

so we have that 

`𝐱 ⋅ 𝐟 ⨲ 𝐮 = -f v`

and

`𝐲 ⋅ 𝐟 ⨲ 𝐮 = f u`.

Reply:
Yes, and I think if they're in the source terms (right hand side) then it's `fv` and `-fu`. So I'm going with `fv` and `fu` as function names.

It's unfortunate that we can't use `×` in function names.